src/index.html

```diff
@@ -1,12 +1,311 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="home-page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="img-link">
+        <img src="./images/logo.png" alt="MoYo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#phone"  class="nav__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+              data-qa="nav-hover"
+              href="#computer"
+              class="nav__link"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gagets" class="nav__link">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#planshet" class="nav__link">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main-content">
+      <div class="main-content__wrap">
+        <div class="card" data-qa="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="Моноблок APPLE A1419 iMac
+          27 Retina 5K (MNED2UA/A)">
```

Comment:
Use shorter `alt` attribute
```suggestion
          <img
            class="card__img"
            src="./images/imac.jpeg"
            alt="Моноблок APPLE A1419"
          >
```

Reply:
Ready!